lib/rocket_chat/server.rb

```diff
@@ -45,5 +45,9 @@ def login(username, password)
       )
       Session.new self, Token.new(response['data'])
     end
+
+    def request_json(path, options = {})
+      super(path, @options.merge(options))
```

Comment:
Nice catch @reist 👍 

Only caveat would be that this is a shallow merge and as such there could be some unexpected results, if say for example you wanted to add some extra property to the `body` ?

Having said that, I'd say this solution is fine, especially for solving the problem at hand